lib/fs_utils/asset.js

```diff
@@ -15,20 +15,19 @@ const logger = require('loggy');
 // => 'app/assets/'
 // Returns String.
 const getAssetDirectory = (path, convention) => {
-  const sep = sysPath.sep;
   const split = path.split('/');
 
   // Creates list similar to:
   // ['app/', 'app/assets/', 'app/assets/thing/', 'app/assets/thing/item.html/']
   return split.map((part, index) => {
-    return split.slice(0, index).concat([part, '']).join(sep);
+    return split.slice(0, index).concat(part, '').join(sysPath.sep);
```

Comment:
`sysPath` will have fast properties and `sysPath.sep` will be loop invariant, so no.